gdx/src/com/badlogic/gdx/scenes/scene2d/ui/ScrollPane.java

```diff
@@ -99,6 +99,18 @@ public ScrollPane (@Null Actor widget, ScrollPaneStyle style) {
 		setActor(widget);
 		setSize(150, 150);
 
+		addListeners();
+	}
+
+	/** Adds all the default listeners. */
+	private void addListeners() {
+		addCaptureListener();
+		this.flickScrollListener = getFlickScrollListener();
+		addListener(this.flickScrollListener);
+		addScrollListener();

```

Comment:
What is the point of this method when it is private and called only from one place? Wouldn't it be simpler to just inline it?

Reply:
I agree, there is no benefit to extracting this to a method.

Reply:
Oh, you are right: that is an artefact of my initial idea of having this method as `protected`.

Will fix!